ansible/slave_libvirt.yml

```diff
@@ -51,7 +51,7 @@
     - name: create a {{ jenkins_user }} user
       user:
         name: "{{ jenkins_user }}"
-        groups: libvirtd
+        groups: libvirt
```

Comment:
since we have a mix of machines that may or may not have this group I think you should check either the distro version or just try both ways ignoring errors